Unity/Assets/YarnSpinner/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "com.thesecretlab.yarnspinner",
+  "version": "1.0.1",
+  "displayName": "Yarn Spinner",
+  "description": "Yarn is a language that's designed to make it super easy to create interactive dialogue for games. Yarn's similar in style to Twine, so if you already know that, you'll be right at home! If you don't, that's cool - Yarn's syntax is extremely minimal, and there's not much there to learn. The Yarn language is used in a number of cool games, including Night In The Woods and Knights and Bikes.",
+  "unity": "2018.1",
+  "keywords": [
+    "Dialogue",
+    "Yarn"
+  ],
+  "author": {
+    "name": "The Secret Lab",
```

Comment:
Change name to "Secret Lab"

Reply:
Done: ac9b44da31cfc95d01bc33ea284ef9135e09933b